tf_encrypted/primitives/sodium/easy_box_test.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=missing-docstring
+import unittest
+
+import numpy as np
+import tensorflow as tf
+
+from tf_encrypted.primitives.sodium import easy_box
+
+
+class TestEasyBox(unittest.TestCase):
+    def setUp(self):
+        tf.reset_default_graph()
+
+    def test_gen_keypair(self):
+        pk, sk = easy_box.gen_keypair()
+        assert isinstance(pk, easy_box.PublicKey), type(pk)
+        assert isinstance(sk, easy_box.SecretKey), type(sk)
+
+        with tf.Session() as sess:
```

Comment:
We will have to validate how easy to integrate this in TFF with TF 1.x version. Wonder if instead we should update this custom ops implementation to TF2.x and have it in separate branch for now. Might be easier to debug TFF with TF2.x.

cc: @jvmncs

Reply:
good point @yanndupis 👍 went with separate distribution for now, `tf-encrypted-primitives`, that only works with TF2 (could easily be made to work for TF1 as well)